tests/test_choose_recipes.py

```diff
@@ -1,14 +1,15 @@
-from rickshaw.choose_recipes import choose_recipes 
-from rickshaw.choose_commodities import choose_commodities
-from rickshaw.choose_niches import random_niches
+from rickshaw.generate import choose_recipes, choose_commodities, random_niches
+from rickshaw.simspec import SimSpec
 
 def test_choose_recipes():
-    niches = random_niches(10)
-    commods = choose_commodities(niches)
+    spec = SimSpec()
+    niches = random_niches(spec, 10)
+    commods = choose_commodities(spec, niches)
     assert type(commods) is list
-    recipes = choose_recipes(commods)
+    recipes = choose_recipes(spec, commods)
     assert type(recipes) is list
-    if "mine" in niches:
+    if "mine" and "enrichment" in niches:
```

Comment:
I don't think that this does what you think it does.  This tests `if len("mine") > 0 and "enrichment" in niches`.  You probably want `if "mine" in niches and "enrichment" in niches`